afteraction-share.html

```diff
@@ -185,6 +193,60 @@ <h2>{{ page.custom_fields.share_slide_title }}</h2>
 		var afteractionDonateAskContent = $('#afteraction-donate-ask-source').html();
 		var afteractionDonateContent = $('#afteraction-donate-source').html();
 
+		const euroCountries = ['Austria', 'Belgium', 'Cyprus', 'Estonia', 'Finland', 'France', 'Germany', 'Greece', 'Ireland', 'Italy', 'Latvia', 'Lithuania', 'Luxembourg', 'Malta', 'Netherlands', 'Portugal', 'Slovakia', 'Slovenia', 'Spain'];
+
+		function updateDonateCurrency(userLocation){
+			if (euroCountries.includes(userLocation)) {
+				userLocation = 'Europe';
+			}
+			var currencyData = {
+				/* uses API exchange data, with hardcoded (out of date) exchange rate as fallback */
+				'Canada': {
+					currencySymbol: '$',
+					currency: 'CAD',
+					exchangeRate: exchangeRateData['CAD'] || 1.45
+				},
+				'Australia': {
+					currencySymbol: '$',
+					currency: 'AUD',
+					exchangeRate:  exchangeRateData['AUD']  || 1.68
+				},
+				'New Zealand': {
+					currencySymbol: '$',
+					currency: 'NZD',
+					exchangeRate:  exchangeRateData['NZD'] || 1.72
+				},
+				'United Kingdom': {
+					currencySymbol: '£',
+					currency: 'GBP',
+					exchangeRate: exchangeRateData['GBP'] || 0.85
+				},
+				'Europe': {
+					currencySymbol: '€',
+					currency: 'EUR',
+					exchangeRate: exchangeRateData['EUR'] || 0.92
+				},
+				'Brazil': {
+					currencySymbol: 'R$',
+					currency: 'BRL',
+					exchangeRate: exchangeRateData['BRL'] || 5.07
+				}
+			};
+			if (currencyData[userLocation]) {
+				$('#afteraction-donate-inner .afteraction-donate-amount-container a').each(function(i){
+					// replace amount and add currency param in donate button urls
+					var $this = $(this);
```

Comment:
I am a sucker for const over var. Could we consider the same here?

Reply:
Good call, thanks Kimani! I've amended in my commit